### PR TITLE
pytorch-mnist.ipynb

### PR DESCRIPTION
**Pull request by ardishakim**

### CODE REVIEW

**Comment by ardishakim** [comment]
Updated hyperparameters for better model accuracy
